### DISCUSSION TITLE
lucky.gen.migration enforce use camel case is necessory?

### DISCUSSION THREAD

**Discussion by zw963** [luckyframework/lucky#1890] [Feature requests & Ideas] [answered]
I often add migration like this:

./bin/lucky.gen.migration add_some_column_to_some_table

This is because i always use like this for ActiveRecord migration, and one more concern, use the camel case in the console is wired.

Do we have any reason to enforce use only camel case?

Thanks.

**Comment by jwoertink** [comment]
I forget why we did that. The name has to be camel case, but since Crystal has a `camelcase` method, I think we add that in

https://github.com/luckyframework/avram/blob/af42b9449a8cccb21632dac3a239d5849428ff9f/src/avram/tasks/gen/migration_generator.cr#L14-L18

These just need to do something like

```
@name = name.camelcase
```

**Reply by zw963** [reply]
Do you consider accept a PR for support underscore form?

Anyway,  `wordsmith` always is the dependency for avram, i consider it quite easy to transform.

**Reply by jwoertink** [reply, marked as answer]
Sure, you can submit a PR for this. :+1:
